finality-aleph/src/justification/backward.rs

```diff
@@ -0,0 +1,44 @@
+use crate::{
```

Comment:
I'm afraid this needs to stay. When a new node enters the network it will get justifications of "ancient" blocks in the `V1` format, hence we need that.